src/builtins/sci-embodied/README.md

```diff
@@ -0,0 +1,111 @@
+
+Read more on [embodied carbon](https://github.com/Green-Software-Foundation/sci/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#embodied-emissions)
+
+## Parameters
+
+### Plugin config
+
+Not Needed
+
+### Inputs
+
+- `device/emissions-embodied`: the sum of Life Cycle Assessment (LCA) emissions for the component
+- `device/expected-lifespan`: the length of time, in seconds, between a component's manufacture and its disposal
+- `resources-reserved`: the number of resources reserved for use by the software
+- `resources-total`: the total number of resources available
+- `duration`: the amount of time covered by an observation, in this context it is used as the share of the total life span of the hardware reserved for use by an application, in seconds.
+
+> Note that if you have a plugin pipeline that adds `vcpus-allocated` and `vcpus-total` to each observation, such as the `cloud-metadata` plugin, those values will be used **in preference** to the given `resources-reserved` and `resources-total` fields.
+
+## Returns
+
+- `carbon-embodied`: the carbon emitted in manufacturing and disposing of a component, in gCO2eq
+
+## Calculation
+
+To calculate the embodied carbon, `m` for a software application, use the equation:
+
+```
+m = te * ts * rs
+```
+
+Where:
+
+- `device/emissions-embodied` = Total embodied emissions; the sum of Life Cycle Assessment (LCA) emissions for the component.
+
+- `timeShare` = Time-share; the share of the total life span of the hardware reserved for use by an application.
+
+  - `timeShare` is calculated as `duration/'device/expected-lifespan'`, where:
+    - `duration` = the length of time the hardware is reserved for use by the software.
+    - `device/expected-lifespan` = Expected lifespan: the length of time, in seconds, between a component's manufacture and its disposal.
+
+- `resourceShare` = Resource-share; the share of the total available resources of the hardware reserved for use by an application.
+  - `resourceShare` is calculated as `resources-reserved/resources-total`, where:
+    - `resources-reserved` = Resources reserved; the number of resources reserved for use by the software.
+    - `resources-total` = Total Resources; the total number of resources available.
+
+## Implementation
+
+IF implements the plugin based on the logic described above. To run the plugin, you must first create an instance of `SciEmbodied`. Then, you can call `execute()` to return `m`.
+
+## Usage
+
+The following snippet demonstrates how to call the `sci-embodied` plugin from Typescript.
+
+```typescript
+import {SciEmbodied} from '@grnsft/if-plugins';
```

Comment:
not imported from `@grnsft/if-plugins` anymore

Reply:
```suggestion
import {SciEmbodied} from 'builtins';
```